packages/side-nav/test/side-nav-item.test.js

```diff
@@ -0,0 +1,249 @@
+      </vaadin-side-nav-item>`);
+      await nextRender(activeItemWithChildren);
+    });
+
+    it('should be expanded', () => {
+      expect(activeItemWithChildren.expanded).to.be.true;
+    });
+
+    it('should have a toggle button', async () => {
+      expect(activeItemWithChildren._button).to.be.ok;
+    });
+
+    it('should collapse when toggle button is clicked', () => {
+      activeItemWithChildren._button.click();
+      expect(activeItemWithChildren.expanded).to.be.false;
+    });
+
+    it('should expand when toggle button is clicked', () => {
+      activeItemWithChildren._button.click();
+      activeItemWithChildren._button.click();
+      expect(activeItemWithChildren.expanded).to.be.true;
+    });
+  });
+
+  describe('item without children', () => {
+    let itemWithoutChildren;
+
+    beforeEach(() => {
+      itemWithoutChildren = fixtureSync('<vaadin-side-nav-item></vaadin-side-nav-item>');
+    });
+
+    it('should not have a toggle button', () => {
+      expect(itemWithoutChildren._button).to.be.not.ok;
+    });
+  });
+});
+
+describe('active', () => {
+  describe('read-only', () => {
+    it('should retain active state', () => {
+      const activeItem = fixtureSync(`<vaadin-side-nav-item path=""></vaadin-side-nav-item>`);
+      activeItem.active = false;
+      expect(activeItem.active).to.be.true;
+    });
+  });
+
+  describe('passive item', () => {
+    let passiveItem;
+
+    beforeEach(() => {
+      passiveItem = fixtureSync(`<vaadin-side-nav-item path="/another-path"></vaadin-side-nav-item>`);
+    });
+
+    it('should not be expanded', () => {
+      expect(passiveItem.expanded).to.be.false;
+    });
+
+    it('should not be active', () => {
+      expect(passiveItem.active).to.be.false;
+    });
+  });
+
+  describe('active item', () => {
+    let activeItem;
+
+    beforeEach(() => {
+      activeItem = fixtureSync(`<vaadin-side-nav-item path=""></vaadin-side-nav-item>`);
+    });
+
+    it('should be expanded', () => {
+      expect(activeItem.expanded).to.be.true;
+    });
+
+    it('should be active', () => {
+      expect(activeItem.active).to.be.true;
+    });
+  });
+});
+
+describe('prefix', () => {
+  let item;
+  let prefixSlot;
+  let prefixContent;
+
+  beforeEach(async () => {
+    item = fixtureSync(
+      `<vaadin-side-nav-item>
+          <vaadin-icon icon="vaadin:folder-open" slot="prefix"></vaadin-icon>
+        </vaadin-side-nav-item>`,
+    );
+    await nextRender(item);
+    prefixSlot = item.shadowRoot.querySelector('slot[name="prefix"]');
+    prefixContent = item.querySelector('vaadin-icon');
+  });
+
+  it('should have prefix slot', () => {
+    expect(prefixSlot).to.be.ok;
+  });
+
+  it('should have prefix content', () => {
+    expect(prefixContent).to.be.ok;
+  });
+
+  it('prefix slot should contain prefix content', () => {
+    expect(prefixSlot.assignedNodes()).to.contain(prefixContent);
+  });
+});
```

Comment:
The presence of prefix and suffix slots is already ensured by snapshot tests for the Shadow DOM, so these tests can be removed, I guess.